website_snippet_country_dropdown/views/assets.xml

```diff
@@ -11,16 +11,11 @@
     <xpath expr="." position="inside">
         <script type="text/javascript"
                 src="/website_snippet_country_dropdown/static/src/js/website_snippet_country_dropdown.js"/>
+        <script type="text/javascript"
+                src="/website_snippet_country_dropdown/static/src/js/web_tour_country_dropdown.js"/>
```

Comment:
Test tour assets should be added in a demo view to avoid overloading the main frontend.